src/components/icons/StartButtonIcon.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+/**
+ * The proptypes for {@link StartButtonIcon}.
+ */
+export interface StartButtonIconProps {
+  id?: string;
```

Comment:
We don't pass `id` though. I think this is one component that doesn't actually need it, though.

Reply:
See this.